Tasks/AzureFileCopyV6/Utility.ps1

```diff
@@ -244,11 +244,12 @@ function Upload-FilesToAzureContainer
             Write-Output "##[command] & azcopy copy `"$sourcePath`" `"$containerURL`" $sanitizedArguments"
```

Comment:
Updating this line is needed as well, as this code path will impact when "Enable shell tasks arguments parameter validation" is enabled.